src/browser/BrowserService.cpp

```diff
@@ -788,59 +789,66 @@ QList<Entry*> BrowserService::sortEntries(QList<Entry*>& pwEntries, const QStrin
     return results;
 }
 
-bool BrowserService::confirmEntries(QList<Entry*>& pwEntriesToConfirm,
-                                    const QString& url,
-                                    const QString& host,
-                                    const QString& submitUrl,
-                                    const QString& realm,
-                                    const bool httpAuth)
+QList<Entry*> BrowserService::confirmEntries(QList<Entry*>& pwEntriesToConfirm,
+                                             const QString& url,
+                                             const QString& host,
+                                             const QString& submitHost,
+                                             const QString& realm,
+                                             const bool httpAuth)
 {
     if (pwEntriesToConfirm.isEmpty() || m_dialogActive) {
-        return false;
+        return {};
     }
 
     m_dialogActive = true;
     BrowserAccessControlDialog accessControlDialog;
+
     connect(m_dbTabWidget, SIGNAL(databaseLocked(DatabaseWidget*)), &accessControlDialog, SLOT(reject()));
-    accessControlDialog.setUrl(!submitUrl.isEmpty() ? submitUrl : url);
-    accessControlDialog.setItems(pwEntriesToConfirm);
-    accessControlDialog.setHTTPAuth(httpAuth);
+    connect(&accessControlDialog, &BrowserAccessControlDialog::disableAccess, [=](QTableWidgetItem* item) {
```

Comment:
You should only capture the needed variables here as well or at least capture by reference.

Reply:
Changed to `&`